__tests__/e2e/scripts/test_integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e pipefail
+
+# run e2e tests journey
+##################################################
+npx @elastic/synthetics synthetics.journey.ts --no-headless
```

Comment:
```suggestion
npx @elastic/synthetics synthetics.journey.ts $SYNTHETICS_E2E_ARGS
```

Can we make no-headless optional? Otherwise this doesn't work on headless environments (like my test linux box0